source/index.ts

```diff
@@ -83,16 +97,16 @@ Object.defineProperties(ow, {
 		}
 	},
 	create: {
-		value: <T>(labelOrPredicate: BasePredicate<T> | string | undefined, predicate?: BasePredicate<T>) => (value: T) => {
+		value: <T>(labelOrPredicate: BasePredicate<T> | string | undefined, predicate?: BasePredicate<T>) => (value: T, label?: string) => {
```

Comment:
```suggestion
		value: <T>(labelOrPredicate: BasePredicate<T> | string | undefined, predicate?: BasePredicate<T>) => ReusableValidator<T> => {
```

?

Reply:
Your suggestion is not a valid TypeScript code.

Reply:
Uhm, right. I was reviewing this on my phone...